src/scripts/modules/media/editbar/block-publish/modals/block-publish-template.html

```diff
@@ -0,0 +1,18 @@
+<div id="reject-publish-modal" class="modal fade">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <form class="form-horizontal">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+          <h4 class="modal-title">You Cannot Publish</h4>
+        </div>
+        <div class="modal-body">
+          You cannot publish because of the following reasons: {{publishBlockers}}
```

Comment:
That's another way to do it. I don't know if it's necessarily a more appropriate way. A new Handlebars helper will have to be created to format the blockers, or the format helper will need to be modified to use in multiple places.